subnet/kube/kube.go

```diff
@@ -103,7 +104,15 @@ func NewSubnetManager() (subnet.Manager, error) {
 		return nil, fmt.Errorf("error creating network manager: %s", err)
 	}
 	go sm.Run(context.Background())
-	return sm, err
+
+	err = wait.Poll(time.Second, 10*time.Second, func() (bool, error) {
```

Comment:
Somewhat arbitrary. If this is not able to sync, the pod will fail and we'll see a restart count in `kubectl` output. Seemed that would at least make some debugging easier than waiting forever (e.g. in the case it cannot contact apiserver). 10 seconds seemed like a reasonable amount of time to wait on this.